modules/menuItems.js

```diff
@@ -432,6 +432,32 @@ var menuTempl = function(webviews) {
         ]
     })
 
+    let languageMenu =
+    Object.keys(i18n.options.resources).map(lang_code => {
+        menuItem = {
+            label: lang_code,
+            click: function(){
+                let mainWindow = Windows.getByType('main');
+                mainWindow.webContents.executeJavaScript(
+                    `TAPi18n.setLanguage("${lang_code}");`
```

Comment:
You would need to call `ipc.send('backendAction_setLanguage', lang_code);` after this line. This should change the menu language